readthedocs/projects/models.py

```diff
@@ -136,6 +136,7 @@ class Project(models.Model):
                     'DirectoryHTMLBuilder">More info</a>.'))
 
     # Project features
+    # TODO: remove this?
     allow_comments = models.BooleanField(_('Allow Comments'), default=False)
     comment_moderation = models.BooleanField(
```

Comment:
Are this fields part of the comment app?

`allow_comments` and `comment_moderation`.

I'm going to see if the tests explode if I remove those lines.

Reply:
Probably should be removed also. Can you grep the code by `allow_comments` and `comment_moderation` to see where it's used?

Reply:
## Relevant files

Here is used to choose the sphinx builder
https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/doc_builder/backends/sphinx.py#L217-L218

## No relevant files

Just to show on the dj admin
https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/projects/admin.py#L105-L106

Test
https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/rtd_tests/tests/test_builds.py#L136

Here is just a comment
https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/projects/forms.py#L232-L234

On a template to link to the comments section
https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/templates/projects/project_edit_base.html#L28-L30

---

For the comment_moderation there is a template that is very related to the comment model. I think that should be removed without problem, but now I'm curious about the sphinx builder.

Reply:
I remove stuff related to `comment_moderation`, and I see that is very related to `allow_comments`,  but I'm have doubts about this

https://github.com/rtfd/readthedocs.org/blob/aefa1b6067f87e2551d8a0e1254ec545b94008d4/readthedocs/doc_builder/backends/sphinx.py#L217-L218

Reply:
I think we can safely remove that `readthedocs-comments` builder and all the `-comments` builders.